cmplibrary/src/main/java/com/sourcepoint/gdpr_cmplibrary/GDPRConsentLib.java

```diff
@@ -201,6 +201,15 @@ public void onSavePM(GDPRUserConsent u) {
             public void onAction(int choiceType, Integer choiceId) {
                 GDPRConsentLib.this.onAction(choiceType, choiceId);
             }
+
+            @Override
+            public void onBackPressAction() {
+                if (isPmOn){
+                    GDPRConsentLib.this.onAction(2,null);
```

Comment:
Here you should rather use ActionTypesClass instead of hardcoding the action types.

Reply:
Good suggestion